index.html

```diff
@@ -184,40 +183,86 @@ <h2>About Myself</h2>
         </ul>
       </div>
       <button type="button" id="resume-button">Get my resume</button>
-      <section class="skills">
+      <section>
         <div class="skill-list">
           <div id="ul-languages">
             <h5>Languages</h5><span class="chevron"><img src="./images/Icon-Chevron-Down-Larger.png" alt="Drop-down chevron"></span>
             <ul id="lang-list">
               <li id="javascript">
                 <div class="img"><img src="./images/javascript.png" alt="Javascript"></div>
-                <div>Javascript</div>
+                <p>Javascript</p>
               </li>
               <li id="html">
                 <div class="img"><img src="./images/html.png" alt="HTML"></div>
-                <div>HTML</div>
+                <p>HTML</p>
               </li>
               <li id="css">
                 <div class="img"><img src="./images/css.png" alt="CSS"></div>
-                <div>CSS</div>
+                <p>CSS</p>
               </li>
             </ul>
           </div>
           <div id="ul-frameworks">
             <h5>Frameworks</h5><span class="chevron"><img src="./images/Disabled.png" alt="Drop-down chevron"></span>
             <ul>
             </ul>
-            <span class="tab"></span>
           </div>
           <div id="ul-skills">
             <h5>Skills</h5><span class="chevron"><img src="./images/Disabled.png" alt="Drop-down chevron"></span>
             <ul>
             </ul>
-            <span class="tab"></span>
           </div>
         </div>
       </section>
     </section>
   </main>
+  <footer class="contact">
+    <div id="contact-primary-text">
+    <h2>Contact me</h2>
+    </div>
+    <div id="contact-bodytext">
+    <p>If you have an application you are interested in developing, a feature that you need built or a 
+      project that needs coding. I'd love to help with it.</p>
+    </div>
+    <form action="https://formspree.io/f/xzbwzzaj" method="POST">
+      <div>  
+        <label for="form-name">
+          <input 
+          type="text" 
+          id="form-name"
+          name="user-name" 
+          maxlength="30" 
+          placeholder="Enter your name"
+          required>
+        </label>
+      </div>  
+      <div>  
+        <label for="form-mail">
+          <input 
+          type="email" 
+          id="form-mail" 
+          name="user-email"
+          autocomplete="on"
+          placeholder="Enter your E-Mail"
+          pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/"
+          required>
+        </label>
+      </div>
+      <div>  
+        <label for="form-message">
+          <textarea 
+          id="form-message" 
+          name="user-message"
+          maxlength="500" 
+          placeholder="Write your message here"
+          required> Write your message here
```

Comment:
- [ ] I think it's better to remove the text "Write your message here" and keep the placeholder. It's a requirement of this project to use placeholders. Remove also the content between the `<textarea>` and `</textarea>` because the text there is covering the placeholder.
```
          placeholder="Write your message here"
          required></textarea>
```

Reply:
Done